code/scss/content.sass

```diff
@@ -74,13 +74,16 @@ $gh-primary-color: #3879D9
       .tt-dropdown-menu
         margin-top: 6px
         width: 100%
-.site-search.repo-scope
+.site-search.scoped-search
   .awesome-autocomplete
     .twitter-typeahead
       width: 230px
       .tt-dropdown-menu
         width: 360px
-        margin-left: -96px
+.site-search.repo-scope .awesome-autocomplete .twitter-typeahead .tt-dropdown-menu
+  margin-left: -96px
+.site-search.org-scope .awesome-autocomplete .twitter-typeahead .tt-dropdown-menu
+  margin-left: -109px
 
```

Comment:
Seems good to me so far. 
The only bad side of this fix is to use fixed hardcoded values, the better way I'd suggest to do this is to get the actual width via Javascript : `$('.scope-badge').width()`

And to apply it to the `.twitter-typeahead` within a css `calc(100% - <extracted-width>)`.
Or to get the width of `.twitter-typeahead` in pixels & substract the width of the `.scope-badge` from it.

But for a quickfix, that works 🙌